docs/metadata.md

```diff
@@ -63,39 +63,45 @@ The URL of the DCAT object inside the RDF representation is the stable identifer
 The following fields must be included in every DCAT object. Metadata will be visible to all pariticipants in the Trust Framework, and may be visible to anyone on the open web without authentication in an open index.
 
 [dcterms:title](https://www.dublincore.org/specifications/dublin-core/dcmi-terms/terms/title/)
-: Short title for this data set.
+: Short title for this dataset.
 
 [dcterms:publisher](https://www.dublincore.org/specifications/dublin-core/dcmi-terms/terms/publisher/)
 : The URL of the Data Provider's record in the Scheme Directory.
 
 [dcterms:license](https://www.dublincore.org/specifications/dublin-core/dcmi-terms/#license)
-: The URL of a Licence, which must be registered with the Registry. All use of this data source is subject to this Licence.
+: The URL of a Licence. All use of this data source is subject to this Licence. Where a data source is Scheme-conforming, the URL will be registered in the Registry.
+
+`ib1:trustFramework`
+: The URL of the Trust Framework(s) the dataset is assured under.
+
+`ib1:datasetAssurance`
+: The assurance level for this dataset.
```

Comment:
Please add
More information about publishing assured data within a Trust Framework is available on the [How to become an assured data publisher](https://icebreakerone.org/sops/assured-data-publishing/) section of the Icebreaker One website.

We may want to move the IB1 page to this documentation stack at some point. I'll add that in the backlog